spec/druid_spec.rb

```diff
@@ -0,0 +1,470 @@
+describe DruidTools::Druid do
+  let(:fixture_dir) { File.expand_path("../fixtures", __FILE__) }
+  let(:valid_druid) { 'druid:cd456ef7890' }
```

Comment:
`loosely_valid_druid`?  `lax_valid_druid`?  `almost_valid_druid`?  😛 

(fine with it staying as-is, but it might be slightly confusing/misleading at first glance if the name is left unqualified)

_(just copying this over as a single comment, since the GH review hid it as outdated, though i'm not sure why)_

Reply:
from the perspective of druid-tools,  cd456ef7890 is valid (it has been valid up till now!);  it's not strictly valid.  I couldn't come up with a good name either.  not_strictly_valid_druid?   unstrictly_valid_druid?  slightly_valid_druid?  valid_ish_druid?  druid_with_vowel?  

_(because you commented on a single commit?)_